src/torchpme/potentials/inversepowerlaw.py

```diff
@@ -17,6 +18,49 @@ def gamma(x: torch.Tensor) -> torch.Tensor:
     return torch.exp(gammaln(x))
 
 
+# Custom exponential function to have an autograd-compatible version of the exponential
+class CustomE1(torch.autograd.Function):
```

Comment:
Very cool!

I think you can put the comment as the docstring.

Also, a question. Probably you checked carefully, but does this has to be a class or can it be also a function? If there is no way we can also wrap 
`CustomE1.apply(z)` into a function `exp1`.

and if this is a public function we should also add a simple test...